tests/utils.go

```diff
@@ -3838,6 +3838,12 @@ func UpdateClusterConfigValue(key string, value string) {
 	cfgMap.Data[key] = value
 	_, err = virtClient.CoreV1().ConfigMaps(KubeVirtInstallNamespace).Update(cfgMap)
 	Expect(err).ToNot(HaveOccurred())
+
+	Eventually(func() string {
+		cfgMap, err := virtClient.CoreV1().ConfigMaps(KubeVirtInstallNamespace).Get(kubevirtConfig, metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		return cfgMap.Data[key]
+	}).Should(BeEquivalentTo(value))
```

Comment:
Hey, good catch. Do we want to introduce a delay in the Eventually to avoid a busy loop? Also, do we want to add a timeout, and fail if the configMap wasn't updated in a timely way?

Reply:
`Eventually` has defaults, both for the timeout and for the polling interval (so it's not really a busy loop).

As per [gomega doc](https://onsi.github.io/gomega/)s: 

> The default value for the timeout is 1 second and the default value for the polling interval is 10 milliseconds. You can change these values by passing them in just after your function

They seemed enough to me and to the results of (some of) the CI runs  (at least for changing a config map, we have looser values in other cases), but I see no problem in relaxing them a bit.

Reply:
ok, LGTM